caikit/interfaces/nlp/data_model/classification.py

```diff
@@ -110,19 +110,22 @@ class ClassifiedGeneratedTextResult(DataObjectBase):
     """
 
     generated_text: Annotated[Optional[str], FieldNumber(1)]  # The generated text
-    token_classification_results: Annotated[
+    input_token_classification_results: Annotated[
         Optional[List[TokenClassificationResult]], FieldNumber(2)
+    ]  # Token classification results for input text
+    output_token_classification_results: Annotated[
```

Comment:
Any reason to go with flattened structure? Should we move it one level down? like:
```
token_classification_results: {
    input:  ...
    output: ...
}
```